UnitTestFrameworkPkg/UnitTestFrameworkPkgHost.dsc.inc

```diff
@@ -41,8 +41,10 @@
   #
   # MSFT
   #
-  MSFT:*_*_*_CC_FLAGS               = /EHsc
-  MSFT:*_*_*_DLINK_FLAGS            == /out:"$(BIN_DIR)\$(MODULE_NAME_GUID).exe" /pdb:"$(BIN_DIR)\$(MODULE_NAME_GUID).pdb" /IGNORE:4001 /NOLOGO /SUBSYSTEM:CONSOLE /DEBUG /STACK:0x40000,0x40000 /NODEFAULTLIB:libcmt.lib libcmtd.lib
+  # MU_CHANGE [BEGIN] - Allow GoogleTest to Test STATIC Functions
+  MSFT:*_*_*_CC_FLAGS               = /EHsc -D GOOGLETEST_HOST_UNIT_TEST_BUILD=1

```

Comment:
I think this doesn't exist in 202302. This won't break anything without the Base.h change, but if you are anticipating needing to test static functions, you need that change as well.